test/beamforming/testFwdBackConv.jl

```diff
@@ -1,77 +1,41 @@
 using Caesar, RoME, IncrementalInference
-
 using DelimitedFiles
 
-# visualization scripts
-
-using Colors
-# currently need master branch
-using Gadfly
-# tags waiting on METADATA
+# Visualization
+using Colors, Gadfly
 using RoMEPlotting, KernelDensityEstimatePlotting
-using Pkg
-
 
+posFile = joinpath(dirname(pathof(Caesar)),"..","test","testdata","test_array_positions.csv");
+posData = readdlm(posFile,',',Float64,'\n')
 
 
-
-sasframe = 64
-logdir = "/media/data1/data/kayaks/20_parsed/array_positions$(sasframe).csv"  # 85
-posData = readdlm(logdir,',',Float64,'\n')
-
 posData[:,1] .+= 13.0
 posData[:,2] .-= 40.0
 
-posData
-
 # gps covariance
 rtkCov = Matrix(Diagonal([0.1;0.1].^2))
 
-
-
-
 N = 200
 fg = initfg()
 
-addVariable!(fg, :x1, Point2)
-setVal!(fg, :x1, zeros(2,N))
-addVariable!(fg, :x2, Point2)
-setVal!(fg, :x2, zeros(2,N))
-addVariable!(fg, :x3, Point2)
-setVal!(fg, :x3, zeros(2,N))
-addVariable!(fg, :x4, Point2)
-setVal!(fg, :x4, zeros(2,N))
-addVariable!(fg, :x5, Point2)
-setVal!(fg, :x5, zeros(2,N))
-# addVariable!(fg, :x6, Point2)
-# setVal!(fg, :x6, zeros(2,N))
-# addVariable!(fg, :x7, Point2)
-# setVal!(fg, :x7, zeros(2,N))
-# addVariable!(fg, :x8, Point2)
-# setVal!(fg, :x8, zeros(2,N))
-# addVariable!(fg, :x9, Point2)
-# setVal!(fg, :x9, zeros(2,N))
-
-
-addVariable!(fg, :l1, Point2)
+#landmark
+addNode!(fg, :l1, Point2)
```

Comment:
Hi @mc2922 , could we change these to all be `addVariable!(fg, :l1, Point2)` instead?

Reply:
changed landmark and locations in loop